ingest/build.gradle

```diff
@@ -4,9 +4,13 @@ plugins {
 }
 
 dependencies {
-	testCompile group: "org.springframework.boot", name: "spring-boot-starter-test", version: "2.1.5.RELEASE"
```

Comment:
✏️ Could we group the types of dependencies?

Reply:
that seems like a reasonable request